nevergrad/benchmark/experiments.py

```diff
@@ -901,6 +902,53 @@ def realworld(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
                             yield xp
 
 
+@registry.register
+def pyomo(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Pyomo models.
+    TODO: we need a comparison with Pyomo solves, and non-linear/complex variants of this.
+    """
+    seedg = create_seed_generator(seed)
+    optims = ["NaiveTBPSA", "SQP", "Powell", "ScrHammersleySearch", "PSO", "OnePlusOne",
+              "NGO", "Shiwa", "DiagonalCMA", "CMA", "TwoPointsDE", "QrDE", "LhsDE", "Zero", "RandomSearch",
+              "HaltonSearch", "MiniDE"]
+    if default_optims is not None:
+        optims = default_optims
+    for budget in [25, 50, 100, 200, 400, 800, 1600]:
+        for num_workers in [1, 30]:
+            if num_workers < budget:
+                for algo in optims:
+                    for fu in get_pyomo_list():
+                        xp = Experiment(fu, algo, budget, num_workers=num_workers, seed=next(seedg))
+                        if not xp.is_incoherent:
+                            yield xp
+
+
+@registry.register
+def sequential_translated_pyomo(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
+    """Pyomo models.
+    TODO: we need a comparison with Pyomo solves, and non-linear/complex variants of this.
+    """
+    seedg = create_seed_generator(seed)
+    optims = ["NaiveTBPSA", "SQP", "Powell", "ScrHammersleySearch", "PSO", "OnePlusOne",
+              "NGO", "Shiwa", "DiagonalCMA", "CMA", "TwoPointsDE", "QrDE", "LhsDE", "Zero", "RandomSearch",
+              "HaltonSearch", "MiniDE"]
+    if default_optims is not None:
+        optims = default_optims
+    at_least_once = False
+    for fu in get_pyomo_list():
+        for budget in [25, 50, 100, 200, 400, 800, 1600]:
+            translated_fu = fu.copy()
```

Comment:
```suggestion
    for budget in [25, 50, 100, 200, 400, 800, 1600]:
        for fu in generate_pyomo_functions():
```
This way, `translated_fu` is not useful anymore